sdk/include/reactos/stubs.h

```diff
@@ -9,24 +31,30 @@
 #define EXCEPTION_WINE_STUB     0x80000100
 #define EH_NONCONTINUABLE       0x01
 
-NTSYSAPI
-VOID
-NTAPI
+/* __int128 is not supported on x86, so use a custom type */
```

Comment:
Isn't it possible to keep this definition within the spec2def generator, so as to keep "locality" of this MyInt128 thing?
(By the way, see commit bd27b050fcd44d38cfbbe4d054dffecfc46c2432 for why this has been introduced.)

Reply:
Why? The whole point of stubs.h is to provide the definitions for the autogenerated stubs file. And that's it. Yes, it's also used in some manually written stubs files, but hopefully nobody will ever include it into anything else. So why duplicate this into every single stubs file?

Reply:
Then let's go with this, I guess...